pkger/service_test.go

```diff
@@ -75,12 +78,12 @@ func TestService(t *testing.T) {
 			})
 
 			t.Run("single bucket new", func(t *testing.T) {
-				testfileRunner(t, "testdata/bucket", func(t *testing.T, pkg *Pkg) {
+				testfileRunner(t, "testdata/bucket.json", func(t *testing.T, pkg *Pkg) {
```

Comment:
this makes it only run against the `.json` version of the file, we support both encodings, but complete coverage is only needed at the parser level, the service is a layer removed from that